src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportProcessingJob.cs

```diff
@@ -94,9 +95,16 @@ record = exportJobRecord;
                     // For single threaded jobs this section will complete a job after every page of results is processed and make a new job for the next page.
                     // This will allow for saving intermediate states of the job.
 
-                    var definition = jobInfo.DeserializeDefinition<ExportJobRecord>();
-                    definition.Progress = exportJobRecord.Progress;
-                    await _queueClient.EnqueueAsync(QueueType.Export, innerCancellationToken, jobInfo.GroupId, definitions: definition);
+                    record.Status = OperationStatus.Completed;
```

Comment:
The handler for `JobSegmentCompletedException()` sets the completion status, do we need this here?

Reply:
I added it here in case we hit an exception later. Once the follow up job is created we don't want the code to think the current job is still running. It is probably redundant, but I don't think it hurts anything.